### DISCUSSION TITLE
WP-Rig Menu Setup

### DISCUSSION THREAD

**Discussion by Fix-Johnson** [wprig/wprig#723] [Q&A]
I have just installed WP-Rig on Local Testing and I cannot figure out how to get my menu, which was originally created with Dive Theme to operate properly.  I just get a html list of links.

Can anyone steer me in the right direction to learn how to fix menu issue?

Greatly appreciated

**Comment by milesdame** [comment]
Hey @Fix-Johnson can you give me some more info? I'm happy to take a stab at helping you.

**Comment by dthenley** [comment]
@Fix-Johnson it sounds like you are wanting your menu to look like it did with your old theme, this is possible but you will need to create it your self. WPRig is a great tool for a lot of customizations but it doesn't have a lot of out of the box styling which is an awesome place to start from scratch. If you need any help just let us know.
